CHANGELOG.md

```diff
@@ -119,6 +119,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 | `--validationSkipDuplicateCheck` |                                   |
 | `--validationS3PathCheck`        |                                   |
 | `--monochromeLogs`               | `--monochrome_logs`               |
+| `skip_short_variant_calling`     | `skip_snv_calling`                |
+| `skip_assembly_wf`               | `skip_genome_assembly`            |
+| `skip_mapping_wf`                | `skip_alignment`                  |
+| `skip_methylation_wf`            | `skip_methylation_analysis`       |
+| `skip_phasing_wf`                | `skip_phasing`                    |
+| `variant_caller`                 | `snv_caller`                      |
+| `parallel_snv`                   | `snv_calling_processes`           |
+| `cadd_prescored`                 | `cadd_prescored_indels`           |
+| `snp_db`                         | `echtvar_snv_databases`           |
+| `variant_catalog`                | `stranger_repeat_catalog`         |
+| `bed`                            | `target_bed`                      |
+| `hificnv_xy`                     | `hificnv_expected_xy_cn`          |
+| `hificnv_xx`                     | `hificnv_expected_xx_cn`          |
+| `hificnv_exclude`                | `hificnv_excluded_regions`        |
```

Comment:
I think both deepvariant and dipcall refer to them as PAR regions, so I'd prefer keeping it that way. Even though technically it's redundant (Swedish CD-skiva is another example). 

But I think we could have `target_regions` instead of `target_bed`.